tests/ui/regions/explicit-static-bound-on-trait.stderr

```diff
@@ -0,0 +1,32 @@
+error[E0478]: lifetime bound not satisfied
+  --> $DIR/explicit-static-bound-on-trait.rs:2:12
+   |
+LL |     value: Box<dyn std::any::Any + 'a>,
+   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+note: lifetime parameter instantiated with the lifetime `'a` as defined here
+  --> $DIR/explicit-static-bound-on-trait.rs:1:14
+   |
+LL | struct Hello<'a> {
+   |              ^^
+note: but lifetime parameter must outlive the static lifetime
+  --> $SRC_DIR/core/src/any.rs:LL:COL
```

Comment:
Locally this actually show the right span, not sure why the stderr doesn't show it correctly.

Reply:
Probably has to do with the sorting of the entries. Try to put the `Option<Ty>` first?

Reply:
Oh, no, because we strip standard library sources in UI tests.